src/encoded/static/components/report.js

```diff
@@ -936,5 +952,7 @@ Report.contextTypes = {
     navigate: PropTypes.func,
 };
 
+export default PageLimitSelector;
```

Comment:
I usually think of the default export as the file’s purpose, but `PageLimitSelector` is more of a utility. I think putting it in its own file now that it’s shared would make sense, and default export it from there. Though with encoded in maintenance mode, this isn’t likely to cause future issues either.